code/object/objcollide.cpp

```diff
@@ -429,6 +429,8 @@ int cpls_aux(vec3d *goal_pos, object *objp2, object *objp)
 		return 0;
 }
 
+//  Maybe consider PROP here too
```

Comment:
If you still wanted to do this, I might simply suggest a `big` local bool, either big|huge for ships or >500 radius for props. It's arbitrary, but I used 500.0f earlier in `collide_prop_weapon`; maybe a constant? `BIG_PROP`?